src/ui/ContextMenu/items/OperatorMenuItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement, ReactNode, useCallback, useRef, useState } from 'react';
+import { BaseChannel, Role, User } from '@sendbird/chat';
+import { MenuItem } from '..';
+import { Member } from '@sendbird/chat/groupChannel';
+import { OpenChannel } from '@sendbird/chat/openChannel';
+
+export interface OperatorMenuItemProps {
+  channel: BaseChannel;
+  user: User & { isMuted: boolean };
+  className?: string | Array<string>;
+  children: ReactElement | ReactElement[] | ReactNode;
+  disable?: boolean;
+  dataSbId?: string;
+  onChange?: (channel: BaseChannel, user: User, isOperator: boolean) => void;
+  onError?: (reason: any) => void;
+}
+
+export const OperatorMenuItem = ({
+  channel,
+  user,
+  className = '',
+  children,
+  disable = false,
+  dataSbId = '',
```

Comment:
@HoonBaek do you know the purpose of `dataSbId`?

Reply:
@chrisallo When components are subdivided for specific purposes, I thought it would be good for `dataSbId` to also be included in each component. It would be good to decide based on the purpose.
Currently, I'm not sure what the purpose of `dataSbId` is.

Reply:
It's for testing,
It's useful when we or customers test the components.

@bang9

Reply:
thank you!